docs/source/reference/storages.rst

```diff
@@ -3,11 +3,12 @@
 optuna.storages
 ===============
 
-The :mod:`~optuna.storages` module defines a :class:`~optuna.storages.BaseStorage` class which abstracts a backend database and provides library-internal interfaces to read/write histories of studies and trials. Library users who wish to use storage solutions other than the default in-memory storage should use one of the child classes of :class:`~optuna.storages.BaseStorage` documented below.
+The :mod:`~optuna.storages` module defines a :class:`~optuna.storages.BaseStorage` class which abstracts a backend database and provides library-internal interfaces to read/write histories of studies and trials. Library users who wish to use storage solutions other than the default in-memory storage should instantiate one of the child classes of :class:`~optuna.storages.BaseStorage` documented below by `~optuna.storages.get_storage`.
```

Comment:
```suggestion
The :mod:`~optuna.storages` module defines a :class:`~optuna.storages.BaseStorage` class which abstracts a backend database and provides library-internal interfaces to read/write histories of studies and trials. Library users who wish to use storage solutions other than the default in-memory storage should instantiate one of the child classes of :class:`~optuna.storages.BaseStorage` documented below. You can instantiate a storage by `~optuna.storages.get_storage`.
```

[IMO] It's not a strong opinion but how about dividing a sentence into two passages?